src/components/code-snippet/__tests__/code-snippet-test-cases.js

```diff
@@ -0,0 +1,50 @@
+import CodeSnippet from '../code-snippet';
+import { highlightHtml } from '../../highlight/html';
+
+const testCases = {};
+const noRenderCases = {};
+
+testCases.everything = {
+  component: CodeSnippet,
+  description: 'A CodeSnippet with every feature (edit buttons, title)',
+  props: {
+    code: `<h1>hi</h1>`,
+    highlighter: () => highlightHtml,
+    filename: 'index.html',
+    edit: {
+      html: '<h1>gurd murn!</h1>',
+      css: 'h1 {color: red}',
+      js: "console.log('hurn')",
```

Comment:
Trying to wrap my head around a real world example here. So if we were to use this with a tutorial code block, we could import the `code` and then use the `extractor` helper function to get the `edit` contents? Like:

```
<CodeSnippet
  code={rawCode}
  highlighter={() => highlightHtml}
  edit={extractor(code)}
  ...
/>
```

Reply:
I'll add a test case!

Reply:
@colleenmcginnis added in https://github.com/mapbox/dr-ui/pull/197/commits/571610ddb2a221ec0abbbe51768c756e1842d426